docs/hardware/binos_outdoor_air_monitor.md

```diff
@@ -0,0 +1,279 @@
+| ------------- | --------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Power Issue   | > Check whether the monitor is plugged<br/>> Check the solar panels                                 | > Check whether the cable from the panels is clearly plugged in the monitor. Plug the monitor if unplugged <br/>> Clean the solar panels using soft cloth or squeegee to clean the upper surface |
+| Communication | > Sim-card out of data<br/>> Sim-card non functional/ cut off by the provider<br/>> GSM malfunction | > Load data if local sim-card used <br/>> Reach out to AirQo <br/>> Listen to the sound made by the monitor to know whether it is a GSM issue                                                    |
+| Sensor Issue  | > No data being reported by the sensors ( PM levels = 0.0)                                          | > Replace the sensors                                                                                                                                                                            |
+
+(ii) Incorrect/inconsistent data
+| Issue | Identification of the issue |Solution|
+| ----------- | ----------- |----------- |
+| Communication | > Monitor taking long to post (possible cause is low battery and the monitor will be making bulk updates) |> Leave the monitor to fully charge and re-evaluate |
+|Sensor Issue |> Low intra sensor correlation (< 0.98)<br/>> Sensor reporting low or higher data than expected |> Replacing the sensors<br/>> Blowing dust off the sensor inlets |
+
+<b>Guide to maintaining the AirQo Monitor</b>
+<b>Power issues</b>
+
+- Use a soft cloth or squeegee to clean the upper surface of the solar panel
+- Check whether the cable from the panels is clearly plugged in the monitor.
+- Replace the batteries if issue is not resolved
+
+![Image]()
+![Image]()
+
+<b>Replacing the Sensors and battery</b>
+![Image]()
+![Image]()
+
+# Data Access
+
+## 4.1 Joining the AirQo platform
+
+To set up an account on the AirQo platform, follow this link: https://platform.airqo.net.
+You will be presented with a dashboard like pictured below.
+
+![Image]()
+
+Air quality monitoring location selection often requires several site scoping
+considerations to be undertaken. These include;
+
+![Image]()
+
+After filing in the required details, click REQUEST to register. Upon registration, you
+will receive an email as illustrated below with access requested and received
+confirmation.
+
+![Image]()
+
+After approval by the AirQo, you will receive a followup email on the next steps which
+include access details with temporary password and illustration on how to set a new
+password to your net-manager account.
+
+## 1.1 Platform Features
```

Comment:
![Screenshot from 2022-11-07 18-55-49](https://user-images.githubusercontent.com/65954740/200355397-b6346904-fb49-406e-a723-7c0dae46abb6.png)
@123MwanjeMike according to the [AirQo Monitor guide](https://drive.google.com/file/d/1LotNVXHDYedjNEN_V8QQA6bY-i-CAE5-/view?usp=sharing) when it reaches Platform Features, the numbering goes to `1.1` instead of `4.1.1` or `4.2`, please guide on how to go about this

Reply:
@NsubugaAnold / @deogratius2phantom , could you provide some guidance on this too?

Reply:
This has been addressed in my recent push @HassanBahati